templates/default/slurm/slurm.conf.erb

```diff
@@ -24,10 +24,10 @@ ReconfigFlags=KeepPartState
 #
 # CLOUD CONFIGS OPTIONS
 <% if node['cfncluster']['use_private_hostname'] == 'true' or node['cfncluster']['cfn_dns_domain'].nil? or node['cfncluster']['cfn_dns_domain'].empty? -%>
-SlurmctldParameters=idle_on_node_suspend
+SlurmctldParameters=idle_on_node_suspend,power_save_min_interval=30
```

Comment:
isn't 30 seconds a little bit too much? what was the default before? I would set it to something like 10 seconds maybe for now? This way we are still able to batch sequence of jobs submitted programmatically and it might not be so impactful for the scaling time e for users that submit jobs individually to leverage the current all-or-nothing approach.

Reply:
also in the changelog you mention `cloud_reg_addrs` but I don't see it anywhere

Reply:
Thanks for the explanation. I still think a worst case scenario of 30 secs to wait after job submission is a bit too much. I would cut it in half unless we are already aware of bottlenecks that might justify the 30 secs value.